src/pages/placeorder.tsx

```diff
@@ -119,19 +119,26 @@ function PlaceOrder() {
             </Typography>
             <TableContainer>
               <Table>
-                {summaryArr.map((el) =>
-                  el[1] ? (
-                    el[0] === 'dropoffDate' || el[0] === 'pickupDate' ? (
+                {formEntriesKeyValueArray.map((formFieldKeyValueArr) =>
```

Comment:
This whole file is quite big already, we need to see which elements can be extracted to get better code readability and control